NAMESPACE

```diff
@@ -9,6 +9,7 @@ export(deal_pipelines)
 export(deal_properties)
 export(deal_properties_history)
 export(deal_stages_history)
+export(default_ld_hubspot_app)
```

Comment:
I added the possibility to "bring your own app" after all, because it wasn't that difficult and will allow for apps with less scopes which sounds more secure.